jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -376,6 +403,33 @@ public StackTraceElement toStackTraceElement() {
 					lineNumber);
 		}
 
+		/*[IF Java10]*/
+		/**
+		 * Answer with a MethodType object for the method associated with this frame.
```

Comment:
This reads awkwardly. Maybe "Return a MethodType describing the stackframe's method"?

Reply:
Changed.  Let me know if you want further changes.